nmn.Templates/README.md

```diff
@@ -109,5 +110,28 @@ The `*|TITLE|*`, `*|START|*` and `*|END|*` can be mixed with whatever markdown c
 
 You can also place  `{{matchingEvents()}}` or `{{listMatchingEvents()}}` in Templates in a similar way, and similar customisation is possible. However, it is defined in a different way, using the matches and template strings defined in the \_configuration file's `addMatchingEvents` array, as shown above.
 
+### weekDates - ouput the beginning and end span/dates of the current week (e.g. Mon-Sun)
```

Comment:
I see a lot of information in the `nmn.Templates/README.md` but there is no associated `/qtn` in this PR so not sure what this is references.